### PR TITLE
Fix: Fix padding and alignment elements on quiz session header

### PR DESCRIPTION
**Pull request by rbacordio**
- Fix horizontal padding on question counter
- Center question set name
- Add question set counter on Header

### CODE REVIEW

File: WebApp/frontend/quiz_session/app/room/quiz/components/header.js [hunk 1]
```diff
@@ -51,10 +53,10 @@ export default function Header() {
         </div>
         <div className="flex justify-center">
           <div className="justify-center items-center">
-            <p className="text-white text-sm mb-2">
-              {metadata?.currentSetName}
+            <p className="text-white text-sm mb-2 text-center">
+              {`(${metadata?.currentSetIndex}/${metadata?.totalNumberOfSets}) ${metadata?.currentSetName}`}
```
Comment (jaymar921) [inline WebApp/frontend/quiz_session/app/room/quiz/components/header.js:55]: ltgm

File: WebApp/frontend/quiz_session/app/room/quiz/components/header.js [hunk 2]
```diff
@@ -22,6 +22,8 @@ export default function Header() {
     { volume: isMute ? 0 : volume / 100 }
   );
 
+  console.log(metadata);
```
Comment (jaymar921) [inline WebApp/frontend/quiz_session/app/room/quiz/components/header.js]: maybe remove this in the future

**Review by jaymar921 — APPROVED** [review]
approved

**Review by kinloveko — APPROVED** [review]
LGTMAY!

**Pull request merged**